main/ZgatewayRF.ino

```diff
@@ -29,6 +29,17 @@
 
 #ifdef ZgatewayRF
 
+#  if defined(ESP32) || defined(ESP8266) || defined(__AVR_ATmega2560__) || defined(__AVR_ATmega1280__)
+#    include <vector>
+using namespace std;
```

Comment:
We have a difference between your macro segmentation and the one in ZgatewayBT:
```
#  if !defined(ESP32) && !defined(ESP8266)
#    include <ArduinoSTL.h>
#  endif

#  include <vector>
using namespace std;
```

It could be interesting to unify both as we are talking about the same function.

Reply:
Right, this one is more concise and shorter. Will do.